Cue/app/tabs/library/DeckView.android.js

```diff
@@ -98,28 +98,39 @@ class DeckView extends React.Component {
       onPress: () => { this.props.navigator.pop() }
     }
 
-    const rightItems = [
+    let rightItems = [
       {
         title: 'Filter flagged cards',
         icon: this.state.isFiltering ? CueIcons.filterToggleSelected : CueIcons.filterToggle,
         onPress: () => this.setState({
           isFiltering: !this.state.isFiltering
         })
-      },
-      {
-        title: 'Add card',
-        icon: CueIcons.plus
-      },
-      {
-        title: 'Deck sharing options',
-        icon: CueIcons.share,
-      },
-      {
-        title: 'Edit',
-        icon: CueIcons.edit
-      },
+      }
     ]
 
+    if (this.props.deck.accession === 'private') {
+      rightItems = rightItems.concat([
+        {
+          title: 'Add card',
+          icon: CueIcons.plus
+        },
+        {
+          title: 'Deck sharing options',
```

Comment:
I think it's better to keep it consistent and leave the sharing options accessible in all cases, but selectively disable functionality based on the sharing status of the current deck. Plus, keeping it always accessible also always keeps the play button in the iOS toolbar centre-aligned in all cases. 😜